### ISSUE TITLE
Class 'Pumpify' incorrectly implements interface 'Duplexify'

### ISSUE THREAD

**Issue by owenniblock** [mafintosh/pumpify#9]
Using pumpify@1.5.1 and duplexify@3.6.1

Running `npm run build` gives the following error:
```
node_modules/@types/pumpify/index.d.ts(12,15): error TS2420: Class 'Pumpify' incorrectly implements interface 'Duplexify'.
  Property 'cork' is missing in type 'Pumpify'.
```

I believe the error may actually lie in https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/pumpify/index.d.ts but may also be a version mismatch somewhere. (Let me know if this Issue should be moved to the types repo).

**Comment by owenniblock** [comment]
Maybe as a result of this change on `@types/duplexify`? https://github.com/DefinitelyTyped/DefinitelyTyped/commit/fee7e62da8fe71e60db027e1e52d480641789b24#diff-2cdac5dad065aa6eb15c23350ea7815b

**Comment by owenniblock** [comment]
Rolling back the `@types/duplexify` works around the issue for now:
> npm install @types/duplexify@3.5.0

**Comment by mafintosh** [comment]
Assuming this got fixed :)

**Comment by owenniblock** [comment]
Sorry for the late reply, I only just had time to check. Yes - this is now resolved :) Looks like `@types/duplexify@3.6.1` disappeared.
